examples/manifold/plot_lle_digits.py

```diff
@@ -116,7 +115,8 @@ def plot_embedding(X, title=None):
 X2 = X.copy()
 X2.flat[::X.shape[1] + 1] += 0.01  # Make X invertible
 t0 = time()
-X_lda = discriminant_analysis.LinearDiscriminantAnalysis(n_components=2).fit_transform(X2, y)
+X_lda = discriminant_analysis.LinearDiscriminantAnalysis(n_components=2
```

Comment:
I see that there is a bit of inconsistency originally in the example. Could you do make sure that we have the following pattern for all the example:

```python
estimator = module.Class(...)
t0 = time()
estimator.fit_transform(...)
```

It will be nicer style for the line breaking then/

Reply:
Actually, let it as it is. We will open a new issue/PR to solve these inconsistencies.